router/middleware/header.go

```diff
@@ -57,26 +57,24 @@ func Secure(c *gin.Context) {
 // information into the request so it will be logged. This is
 // intended for debugging and troubleshooting.
 func RequestVersion(c *gin.Context) {
-	apiVersion := version.Version
+	v := version.New()
 
 	if gin.Mode() == "debug" {
-		c.Request.Header.Set("X-Vela-Version", apiVersion.String())
+		c.Request.Header.Set("X-Vela-Version", v.Semantic())
 	} else { // in prod we don't want the build number metadata
-		apiVersion.Metadata = ""
-		c.Request.Header.Set("X-Vela-Version", apiVersion.String())
+		c.Request.Header.Set("X-Vela-Version", v.Semantic())
 	}
 }
 
 // ResponseVersion is a middleware function that injects the Vela API version
 // information into the response so it will be logged. This is
 // intended for debugging and troubleshooting.
 func ResponseVersion(c *gin.Context) {
-	apiVersion := version.Version
+	v := version.New()
 
 	if gin.Mode() == "debug" {
-		c.Header("X-Vela-Version", apiVersion.String())
+		c.Header("X-Vela-Version", v.Semantic())
 	} else { // in prod we don't want the build number metadata
-		apiVersion.Metadata = ""
-		c.Header("X-Vela-Version", apiVersion.String())
+		c.Header("X-Vela-Version", v.Semantic())
```

Comment:
maybe i missed something, but do we need this if statement if we're returning the same thing?

Reply:
No we don't need the if statement 👍 

If you'd like I can comment out the `if` but I left it in favor of being explicit on how we could customize it in the future.

In short, my thought process was by leaving it there, if (or when) we want to look at customizing how that header is populated from the API when running in `debug` mode then the shell of the code is there.

Reply:
K, not a big deal to leave it in 👍